src/org/etools/j1939_84/controllers/part03/Part03Step16Controller.java

```diff
@@ -54,12 +58,41 @@
     @Override
     protected void run() throws Throwable {
         // 6.3.16.1.a. Turn the engine off.
+        incrementProgress("Part 3, Step 16 Turn Engine Off and keep the ignition key in the off position");
+        ensureKeyOffEngineOff();
+
         // 6.3.16.1.b. Confirm Fault A is still implanted according to the manufacturer’s instruction.
+        incrementProgress("Waiting for implant of Fault A according to the engine manufacturer's instruction");
+        waitForFault("Part 6.3.16.1.b");
+
         // 6.3.16.1.c. Wait manufacturer’s recommended interval with the key in the off position.
+        incrementProgress("Waiting for manufacturer's recommended interval after the Fault A implant");
```

Comment:
This appears to be copy pasted from the previous step.

Reply:
What would you like this to say?

Reply:
Perhaps "Part 3, Step 16: Waiting manufacturer’s recommended interval with the key in the off position."  This will be the message displayed in the status bar while the dialog box is shown.  If they take a screen shot, it should reflect what the app is actually doing.

"Part 3, Step 16" could be "6.3.16.1.c", but I think we should be consistent.. at least going forward from here.